src/Generator/Generators/CSharp/CSharpTextTemplate.cs

```diff
@@ -2581,6 +2581,17 @@ public struct ParamMarshal
             return paramMarshal;
         }
 
+        private CSharpTypePrinterResult ReturnTypeCheckIsParamRefConvertible(Parameter param)
+        {
+            if(!param.HasDefaultValue && !param.IsOut && !param.IsInOut
```

Comment:
Please handle default arguments by generating overloads rather than leaving pointers. See the pass for default arguments.